src/libutil/hash.cc

```diff
@@ -456,4 +458,13 @@ std::string_view printHashAlgo(HashAlgorithm ha)
     }
 }
 
+void to_json(nlohmann::json & json, const Hash & hash)
+{
+    json = nlohmann::json::object(
+        {
+            {"algo", printHashAlgo(hash.algo)},
+            {"base16", hash.to_string(HashFormat::Base16, false)},
+        });
+}
```

Comment:
I think I preferred it when it was consistently emitting SRI-formatted hashes. It was a bit more predictable.

Reply:
(bump)

Reply:
I don't have strong feelings about it, but in JSON it seems more idiomatic to emit a structured representation of the hash (i.e. one where you don't have to parse the hash to figure out the hash algorithm, etc.).